shared/src/test/diff/qq/Basic.mls

```diff
@@ -82,15 +79,11 @@ run(unquoted)
 let str_lit = code""" "hello" """
 //│ let str_lit: Code["hello", nothing]
 
-// TODO support unbound free vars
-:e
+
 let y = 1
 let free = code"y"
-//│ ╔══[ERROR] identifier not found: y
```

Comment:
This is a level error. We don't intend to support cross stage persistence for now. It shouldn't type check.

Reply:
Oh never mind I just saw it's a global binding.

But in that case, you should add a new test case with a *local* y binding to reproduce the original error that this was testing.

Reply:
OK, I now saw that you do have a test like this later on. You can disregard this monologue!